src/apps/chifra/cmd/serve/exec/admin_pins.go

```diff
@@ -0,0 +1,62 @@
+package serveRoutes
```

Comment:
I didn't know you could have a package named differently than the folder it's in. I'm not saying it's wrong, I'm just curious to see it.

Reply:
Yes, as far as I know package name doesn't have to be the same as folder's, but you can only have one package per folder

Reply:
Well, I actually renamed the folder and forgot to rename the package. I will fix it now, thanks for catching it